### ISSUE TITLE
The Other Nefertiti

### ISSUE THREAD

**Issue by derhuerst** [ipfs/archives#71]
http://nefertitihack.alloversky.com/

**Comment by edsilv** [comment]
https://ipfs.io/ipfs/QmY22Pv2h1PpjoAVXUvrBQun4cWyeG1VBuU5WHKgduDGXF :-)

**Comment by eminence** [comment]
I'm trying to download this, edsilv, but I'm stuck at 62.81% -- is your node still running?

**Comment by edsilv** [comment]
Hmm, I asked @davidar to pin it for me, so should be on the IPFS team's
servers...

**Comment by Kubuxu** [comment]
I just re-added it: fs:/ipfs/Qma14MoaXKb6LdBoZShmD9Tt9fH44rNDgLuCbCWHT6GVVQ/

**Comment by eminence** [comment]
The new file has a different hash from the original -- is that expected?

`added QmbWxDZTwwsTc77hTzNLqbB8QEpD4h4AFKHaCtkY5dJh2C NefertitiHack3D.stl`

**Comment by eminence** [comment]
Ohh, I see -- the original file looks like an OBJ file, but this new file is an STL

**Comment by Kubuxu** [comment]
Yeah, I downloaded the torrent, let me create directory with both of those files.
